cpu/stm32/include/periph_cpu.h

```diff
@@ -1174,6 +1174,39 @@ void stm32_eth_common_init(void);
 #define HAVE_PTP_TIMER_SET_ABSOLUTE 1   /**< Native implementation available */
 /** @} */
 
+#if defined(CPU_HAS_BACKUP_RAM) || defined(DOXYGEN)
+/**
+ * @brief   Enable backup RAM access
+ */
+void backup_ram_init(void);
+
+/**
+ * @brief   Check whether the backup RAM regulator is on and backup RAM
+ *          content is retained while asleep
+ */
+bool backup_ram_is_retained(void);
+
+/**
+ * @brief   Enable the backup RAM regulator to retain it´s content
+ *          during standby mode
+ */
+void backup_ram_sleep(void);
+
+/**
+ * @brief   Disable the backup RAM regulator
```

Comment:
But the backup RAM contest is still retained if the CPU is awake right?

Reply:
From the F7 datasheet chapter 4.1.3:
```
When the backup domain is supplied by VDD (analog switch connected to VDD), the backup 
SRAM is powered from VDD which replaces the VBAT power supply to save battery life.
When the backup domain is supplied by VBAT (analog switch connected to VBAT because 
VDD is not present), the backup SRAM is powered by a dedicated low-power regulator. This 
regulator can be ON or OFF depending whether the application needs the backup SRAM 
function in Standby and VBAT modes or not. The power-down of this regulator is controlled 
by a dedicated bit, the BRE control bit of the PWR_CSR1 register (see Section 4.4.2: PWR 
power control/status register (PWR_CSR1)). 
```
It could be that the regulator also must be on if the board is powered from a battery, I was not 100% sure how to understand this paragraph. For the case when the board is powered by USB the regulator is only required when the CPU sleeps. We could play save and enable it all the time but maybe waste power energy or someone with a battery backed board could try `tests/periph_backup_ram`.
Or I could think of a `#define CPU_BACKUP_RAM_REGULATOR_ALWAYS_ON` somewhere.

Reply:
Can you measure the current to be sure what's going on? Often dev boards have a jumper that you can replace with a multimeter to measure the current.

Reply:
Option 1:
You mean I should measure the current when the regulator is on and when it is off?
So if the current does not increase significantly, we keep the regulator enabled all the time?

Option 2:
To really know if the regulator is required when the board is powered only by `V_Bat`, I think I would have to remove solder bridge (156) that connects `V_DD` with `V_Bat` on my `nucleo-f767zi` and then connect a coin cell with `GND` and `V_Bat` pin to power the board.

Reply:
I'm just saying when in doubt, measure :wink: 
Maybe the STM32 hardware is more clever than we give it credit and the regulator will automatically turn off when not in standby mode.

Reply:
## Experiment:
We have an adapter board for the `nucleo-f767zi` that measures current with an `ina3221`.
Run `tests/periph_backup_ram` with `DELAY_SEC=300` seconds (5 minutes) and exit after the delay expired.
A measurement of the current consumption is printed about once a second.
Results for (A) regulator on, and (B) regulator off are averaged and compared.

### A: backup_ram_init() calls backup_ram_regulator_on()

<details open>
<summary>regulator_on.txt</summary>

<p>

```
2021-09-23 18:37:42,966 # powermon,channel=Nucleo-f767zi shunt_uv=45960,bus_mv=5072,current_ua=91920,power_uw=466624
2021-09-23 18:37:44,022 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:37:45,078 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5064,current_ua=91360,power_uw=460824
2021-09-23 18:37:46,134 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:47,189 # powermon,channel=Nucleo-f767zi shunt_uv=51560,bus_mv=5072,current_ua=103120,power_uw=522416
2021-09-23 18:37:48,246 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:49,302 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:50,357 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:51,414 # powermon,channel=Nucleo-f767zi shunt_uv=51680,bus_mv=5072,current_ua=103360,power_uw=522416
2021-09-23 18:37:52,470 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:53,526 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:54,582 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:55,638 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:56,694 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:57,750 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:58,806 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:37:59,862 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:00,918 # powermon,channel=Nucleo-f767zi shunt_uv=49880,bus_mv=5064,current_ua=99760,power_uw=506400
2021-09-23 18:38:01,974 # powermon,channel=Nucleo-f767zi shunt_uv=45720,bus_mv=5072,current_ua=91440,power_uw=461552
2021-09-23 18:38:03,029 # powermon,channel=Nucleo-f767zi shunt_uv=45720,bus_mv=5072,current_ua=91440,power_uw=461552
2021-09-23 18:38:04,086 # powermon,channel=Nucleo-f767zi shunt_uv=45720,bus_mv=5072,current_ua=91440,power_uw=461552
2021-09-23 18:38:05,142 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:06,198 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5080,current_ua=91360,power_uw=462280
2021-09-23 18:38:07,254 # powermon,channel=Nucleo-f767zi shunt_uv=48360,bus_mv=5072,current_ua=96720,power_uw=491984
2021-09-23 18:38:08,310 # powermon,channel=Nucleo-f767zi shunt_uv=45720,bus_mv=5072,current_ua=91440,power_uw=461552
2021-09-23 18:38:09,366 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:10,422 # powermon,channel=Nucleo-f767zi shunt_uv=47600,bus_mv=5064,current_ua=95200,power_uw=481080
2021-09-23 18:38:11,478 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:12,535 # powermon,channel=Nucleo-f767zi shunt_uv=47800,bus_mv=5072,current_ua=95600,power_uw=486912
2021-09-23 18:38:13,590 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:14,646 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:15,702 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:16,758 # powermon,channel=Nucleo-f767zi shunt_uv=49840,bus_mv=5072,current_ua=99680,power_uw=507200
2021-09-23 18:38:17,814 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:18,870 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5080,current_ua=91360,power_uw=462280
2021-09-23 18:38:19,926 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5064,current_ua=91360,power_uw=460824
2021-09-23 18:38:20,982 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:22,036 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:23,092 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:24,150 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:25,204 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:26,262 # powermon,channel=Nucleo-f767zi shunt_uv=52000,bus_mv=5072,current_ua=104000,power_uw=527488
2021-09-23 18:38:27,318 # powermon,channel=Nucleo-f767zi shunt_uv=45720,bus_mv=5072,current_ua=91440,power_uw=461552
2021-09-23 18:38:28,372 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:29,430 # powermon,channel=Nucleo-f767zi shunt_uv=45720,bus_mv=5072,current_ua=91440,power_uw=461552
2021-09-23 18:38:30,487 # powermon,channel=Nucleo-f767zi shunt_uv=45720,bus_mv=5072,current_ua=91440,power_uw=461552
2021-09-23 18:38:31,542 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:32,597 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:33,652 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:34,712 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:35,766 # powermon,channel=Nucleo-f767zi shunt_uv=51720,bus_mv=5064,current_ua=103440,power_uw=521592
2021-09-23 18:38:36,821 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:37,879 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:38,934 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:39,988 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:41,048 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:42,102 # powermon,channel=Nucleo-f767zi shunt_uv=46760,bus_mv=5072,current_ua=93520,power_uw=476768
2021-09-23 18:38:43,158 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:44,212 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:45,270 # powermon,channel=Nucleo-f767zi shunt_uv=48360,bus_mv=5064,current_ua=96720,power_uw=491208
2021-09-23 18:38:46,326 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:47,385 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:48,438 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:49,494 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:50,550 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:51,606 # powermon,channel=Nucleo-f767zi shunt_uv=48840,bus_mv=5072,current_ua=97680,power_uw=497056
2021-09-23 18:38:52,662 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:53,720 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:54,774 # powermon,channel=Nucleo-f767zi shunt_uv=47240,bus_mv=5056,current_ua=94480,power_uw=475264
2021-09-23 18:38:55,830 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:56,886 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:57,942 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:38:58,999 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:00,055 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:01,111 # powermon,channel=Nucleo-f767zi shunt_uv=50520,bus_mv=5072,current_ua=101040,power_uw=512272
2021-09-23 18:39:02,166 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:03,222 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5080,current_ua=91360,power_uw=462280
2021-09-23 18:39:04,279 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5064,current_ua=91360,power_uw=460824
2021-09-23 18:39:05,334 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:06,390 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:07,446 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:08,503 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5040,current_ua=91280,power_uw=458640
2021-09-23 18:39:09,559 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:10,615 # powermon,channel=Nucleo-f767zi shunt_uv=51840,bus_mv=5072,current_ua=103680,power_uw=527488
2021-09-23 18:39:11,671 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:12,727 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:13,783 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:14,839 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:15,895 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:16,951 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:18,007 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:19,064 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:20,119 # powermon,channel=Nucleo-f767zi shunt_uv=51480,bus_mv=5064,current_ua=102960,power_uw=521592
2021-09-23 18:39:21,175 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:22,231 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:23,287 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:24,343 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:25,400 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:39:26,455 # powermon,channel=Nucleo-f767zi shunt_uv=47240,bus_mv=5072,current_ua=94480,power_uw=476768
2021-09-23 18:39:27,511 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:39:28,569 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:29,623 # powermon,channel=Nucleo-f767zi shunt_uv=48280,bus_mv=5064,current_ua=96560,power_uw=491208
2021-09-23 18:39:30,680 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:31,734 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:32,791 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5080,current_ua=91360,power_uw=462280
2021-09-23 18:39:33,847 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:34,903 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:35,959 # powermon,channel=Nucleo-f767zi shunt_uv=49360,bus_mv=5080,current_ua=98720,power_uw=502920
2021-09-23 18:39:37,013 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:38,072 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:39:39,127 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:39:40,183 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:41,239 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:42,296 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5040,current_ua=91280,power_uw=458640
2021-09-23 18:39:43,351 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:39:44,407 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5080,current_ua=91360,power_uw=462280
2021-09-23 18:39:45,463 # powermon,channel=Nucleo-f767zi shunt_uv=51480,bus_mv=5072,current_ua=102960,power_uw=522416
2021-09-23 18:39:46,520 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:47,575 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:48,632 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:49,687 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:50,743 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:51,800 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:52,855 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:53,912 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:54,968 # powermon,channel=Nucleo-f767zi shunt_uv=51200,bus_mv=5064,current_ua=102400,power_uw=516528
2021-09-23 18:39:56,024 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:57,080 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:58,136 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:39:59,192 # powermon,channel=Nucleo-f767zi shunt_uv=49320,bus_mv=5072,current_ua=98640,power_uw=502128
2021-09-23 18:40:00,248 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:01,304 # powermon,channel=Nucleo-f767zi shunt_uv=46200,bus_mv=5072,current_ua=92400,power_uw=466624
2021-09-23 18:40:02,360 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:03,417 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:04,472 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:40:05,528 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:06,584 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:07,640 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:08,697 # powermon,channel=Nucleo-f767zi shunt_uv=47240,bus_mv=5064,current_ua=94480,power_uw=476016
2021-09-23 18:40:09,752 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:40:10,808 # powermon,channel=Nucleo-f767zi shunt_uv=48320,bus_mv=5072,current_ua=96640,power_uw=491984
2021-09-23 18:40:11,865 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:12,920 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:13,976 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:15,032 # powermon,channel=Nucleo-f767zi shunt_uv=50560,bus_mv=5072,current_ua=101120,power_uw=512272
2021-09-23 18:40:16,088 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:17,144 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:18,200 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:40:19,256 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:20,312 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:21,368 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:22,424 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:23,480 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:24,536 # powermon,channel=Nucleo-f767zi shunt_uv=51720,bus_mv=5072,current_ua=103440,power_uw=522416
2021-09-23 18:40:25,593 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:26,648 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:27,704 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:28,760 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:29,816 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:30,872 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:31,929 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:32,984 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:34,040 # powermon,channel=Nucleo-f767zi shunt_uv=51440,bus_mv=5064,current_ua=102880,power_uw=521592
2021-09-23 18:40:35,094 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:36,152 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:37,208 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:40:38,265 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:39,321 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:40,377 # powermon,channel=Nucleo-f767zi shunt_uv=47480,bus_mv=5072,current_ua=94960,power_uw=481840
2021-09-23 18:40:41,433 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:42,488 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:43,546 # powermon,channel=Nucleo-f767zi shunt_uv=48280,bus_mv=5064,current_ua=96560,power_uw=491208
2021-09-23 18:40:44,601 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:45,656 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:40:46,714 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:47,769 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:48,825 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:49,882 # powermon,channel=Nucleo-f767zi shunt_uv=48920,bus_mv=5072,current_ua=97840,power_uw=497056
2021-09-23 18:40:50,937 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:51,991 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:53,049 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:40:54,106 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5080,current_ua=91360,power_uw=462280
2021-09-23 18:40:55,161 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:56,217 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:57,273 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:58,329 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:40:59,386 # powermon,channel=Nucleo-f767zi shunt_uv=51440,bus_mv=5072,current_ua=102880,power_uw=522416
2021-09-23 18:41:00,441 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:01,497 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:02,553 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:41:03,609 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:04,665 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:41:05,721 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:41:06,777 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:07,833 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:08,889 # powermon,channel=Nucleo-f767zi shunt_uv=51600,bus_mv=5072,current_ua=103200,power_uw=522416
2021-09-23 18:41:09,945 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:11,001 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:12,057 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:13,113 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:14,169 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:15,225 # powermon,channel=Nucleo-f767zi shunt_uv=46720,bus_mv=5072,current_ua=93440,power_uw=471696
2021-09-23 18:41:16,281 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:17,338 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:18,394 # powermon,channel=Nucleo-f767zi shunt_uv=48320,bus_mv=5064,current_ua=96640,power_uw=491208
2021-09-23 18:41:19,449 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:20,506 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:21,562 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:22,618 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:41:23,674 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:24,730 # powermon,channel=Nucleo-f767zi shunt_uv=48840,bus_mv=5072,current_ua=97680,power_uw=497056
2021-09-23 18:41:25,786 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:26,841 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:27,898 # powermon,channel=Nucleo-f767zi shunt_uv=46680,bus_mv=5064,current_ua=93360,power_uw=470952
2021-09-23 18:41:28,954 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5080,current_ua=91360,power_uw=462280
2021-09-23 18:41:30,010 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:31,066 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:32,122 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:41:33,178 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:34,234 # powermon,channel=Nucleo-f767zi shunt_uv=50960,bus_mv=5072,current_ua=101920,power_uw=517344
2021-09-23 18:41:35,289 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:36,346 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:37,402 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:38,458 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:39,514 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:40,570 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:41,627 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:42,682 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:43,738 # powermon,channel=Nucleo-f767zi shunt_uv=51480,bus_mv=5064,current_ua=102960,power_uw=521592
2021-09-23 18:41:44,794 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:45,850 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:46,906 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:47,963 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5056,current_ua=91280,power_uw=460096
2021-09-23 18:41:49,018 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:50,074 # powermon,channel=Nucleo-f767zi shunt_uv=45680,bus_mv=5072,current_ua=91360,power_uw=461552
2021-09-23 18:41:51,131 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:52,187 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:41:53,243 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:41:54,299 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:41:55,355 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:56,411 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:57,467 # powermon,channel=Nucleo-f767zi shunt_uv=47240,bus_mv=5064,current_ua=94480,power_uw=476016
2021-09-23 18:41:58,523 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:41:59,579 # powermon,channel=Nucleo-f767zi shunt_uv=47400,bus_mv=5072,current_ua=94800,power_uw=481840
2021-09-23 18:42:00,635 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:01,691 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:02,746 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:03,803 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:04,858 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5040,current_ua=91200,power_uw=458640
2021-09-23 18:42:05,916 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:06,971 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:42:08,027 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:09,084 # powermon,channel=Nucleo-f767zi shunt_uv=50400,bus_mv=5072,current_ua=100800,power_uw=512272
2021-09-23 18:42:10,139 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:11,195 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:12,251 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:13,307 # powermon,channel=Nucleo-f767zi shunt_uv=51680,bus_mv=5072,current_ua=103360,power_uw=522416
2021-09-23 18:42:14,363 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:15,420 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:16,475 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:17,531 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:18,588 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5056,current_ua=91280,power_uw=460096
2021-09-23 18:42:19,643 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:20,699 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:21,755 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:22,811 # powermon,channel=Nucleo-f767zi shunt_uv=50440,bus_mv=5064,current_ua=100880,power_uw=511464
2021-09-23 18:42:23,867 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:24,923 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:25,980 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:27,035 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:28,091 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:29,147 # powermon,channel=Nucleo-f767zi shunt_uv=47760,bus_mv=5072,current_ua=95520,power_uw=486912
2021-09-23 18:42:30,203 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:31,257 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:32,315 # powermon,channel=Nucleo-f767zi shunt_uv=47760,bus_mv=5064,current_ua=95520,power_uw=486144
2021-09-23 18:42:33,371 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:34,427 # powermon,channel=Nucleo-f767zi shunt_uv=47240,bus_mv=5072,current_ua=94480,power_uw=476768
2021-09-23 18:42:35,483 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:36,539 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:42:37,596 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:42:38,652 # powermon,channel=Nucleo-f767zi shunt_uv=50360,bus_mv=5080,current_ua=100720,power_uw=513080
2021-09-23 18:42:39,708 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:42:39,709 # Exiting Pyterm
```

</p>

</details> 

```
$cat regulator_on.txt | tr ',' ' ' | awk '{print $9}' | sed 's/current_ua=//g' | awk '{x+=$0}END{print x/NR}'`
92250.5
```

### B: backup_ram_regulator_on() is never called

<details open>
<summary>regulator_off.txt</summary>

<p>

```
2021-09-23 18:44:21,079 # powermon,channel=Nucleo-f767zi shunt_uv=45920,bus_mv=5072,current_ua=91840,power_uw=466624
2021-09-23 18:44:22,135 # powermon,channel=Nucleo-f767zi shunt_uv=45560,bus_mv=5072,current_ua=91120,power_uw=461552
2021-09-23 18:44:23,191 # powermon,channel=Nucleo-f767zi shunt_uv=45560,bus_mv=5072,current_ua=91120,power_uw=461552
2021-09-23 18:44:24,247 # powermon,channel=Nucleo-f767zi shunt_uv=45560,bus_mv=5064,current_ua=91120,power_uw=460824
2021-09-23 18:44:25,303 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:26,357 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:27,414 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:28,472 # powermon,channel=Nucleo-f767zi shunt_uv=47720,bus_mv=5064,current_ua=95440,power_uw=481080
2021-09-23 18:44:29,527 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:30,583 # powermon,channel=Nucleo-f767zi shunt_uv=46680,bus_mv=5072,current_ua=93360,power_uw=471696
2021-09-23 18:44:31,639 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:32,694 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:33,751 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:34,805 # powermon,channel=Nucleo-f767zi shunt_uv=49280,bus_mv=5072,current_ua=98560,power_uw=502128
2021-09-23 18:44:35,863 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:36,919 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:37,976 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5056,current_ua=91200,power_uw=460096
2021-09-23 18:44:39,032 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:40,088 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:44:41,142 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:42,198 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:43,255 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:44,310 # powermon,channel=Nucleo-f767zi shunt_uv=50920,bus_mv=5064,current_ua=101840,power_uw=516528
2021-09-23 18:44:45,366 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:46,424 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:47,480 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5064,current_ua=91200,power_uw=460824
2021-09-23 18:44:48,535 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:49,591 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:44:50,648 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:51,703 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:44:52,760 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:44:53,815 # powermon,channel=Nucleo-f767zi shunt_uv=51680,bus_mv=5064,current_ua=103360,power_uw=521592
2021-09-23 18:44:54,871 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:44:55,928 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:44:56,984 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:44:58,038 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5048,current_ua=91200,power_uw=459368
2021-09-23 18:44:59,097 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:00,150 # powermon,channel=Nucleo-f767zi shunt_uv=46720,bus_mv=5072,current_ua=93440,power_uw=471696
2021-09-23 18:45:01,208 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:02,264 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:03,320 # powermon,channel=Nucleo-f767zi shunt_uv=48480,bus_mv=5064,current_ua=96960,power_uw=491208
2021-09-23 18:45:04,376 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:05,429 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:06,486 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:07,544 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:45:08,600 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:09,656 # powermon,channel=Nucleo-f767zi shunt_uv=49320,bus_mv=5072,current_ua=98640,power_uw=502128
2021-09-23 18:45:10,712 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:11,768 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:12,824 # powermon,channel=Nucleo-f767zi shunt_uv=46080,bus_mv=5064,current_ua=92160,power_uw=465888
2021-09-23 18:45:13,878 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:14,935 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:45:15,990 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:17,048 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:18,104 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:19,160 # powermon,channel=Nucleo-f767zi shunt_uv=51480,bus_mv=5072,current_ua=102960,power_uw=522416
2021-09-23 18:45:20,216 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:21,272 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:22,328 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:23,384 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:24,441 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:45:25,496 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:26,553 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:27,609 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:28,665 # powermon,channel=Nucleo-f767zi shunt_uv=51320,bus_mv=5072,current_ua=102640,power_uw=522416
2021-09-23 18:45:29,721 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:30,777 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:31,833 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:45:32,889 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:45:33,945 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:45:35,001 # powermon,channel=Nucleo-f767zi shunt_uv=46680,bus_mv=5072,current_ua=93360,power_uw=471696
2021-09-23 18:45:36,057 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:37,113 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:38,169 # powermon,channel=Nucleo-f767zi shunt_uv=48760,bus_mv=5064,current_ua=97520,power_uw=496272
2021-09-23 18:45:39,225 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:40,281 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:45:41,338 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:42,393 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:43,449 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:44,506 # powermon,channel=Nucleo-f767zi shunt_uv=48760,bus_mv=5080,current_ua=97520,power_uw=497840
2021-09-23 18:45:45,562 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:46,617 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:47,673 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:45:48,730 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:45:49,785 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:50,841 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:51,897 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:52,955 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:45:54,009 # powermon,channel=Nucleo-f767zi shunt_uv=51080,bus_mv=5072,current_ua=102160,power_uw=517344
2021-09-23 18:45:55,066 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:45:56,119 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:45:57,178 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:45:58,235 # powermon,channel=Nucleo-f767zi shunt_uv=51000,bus_mv=5080,current_ua=102000,power_uw=518160
2021-09-23 18:45:59,290 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:00,346 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:46:01,401 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:46:02,457 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:46:03,513 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:46:04,569 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:46:05,625 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5064,current_ua=91280,power_uw=460824
2021-09-23 18:46:06,682 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:46:07,738 # powermon,channel=Nucleo-f767zi shunt_uv=48760,bus_mv=5064,current_ua=97520,power_uw=496272
2021-09-23 18:46:08,794 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:09,850 # powermon,channel=Nucleo-f767zi shunt_uv=46680,bus_mv=5072,current_ua=93360,power_uw=471696
2021-09-23 18:46:10,907 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:46:11,962 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:13,018 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:14,074 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:15,130 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:46:16,186 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:17,242 # powermon,channel=Nucleo-f767zi shunt_uv=46120,bus_mv=5064,current_ua=92240,power_uw=465888
2021-09-23 18:46:18,299 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:19,354 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5040,current_ua=91200,power_uw=458640
2021-09-23 18:46:20,411 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:21,467 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:22,520 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:23,579 # powermon,channel=Nucleo-f767zi shunt_uv=51440,bus_mv=5072,current_ua=102880,power_uw=522416
2021-09-23 18:46:24,635 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:46:25,691 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:26,747 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:27,803 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:46:28,859 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:46:29,915 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:46:30,971 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:46:32,027 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:33,082 # powermon,channel=Nucleo-f767zi shunt_uv=51520,bus_mv=5064,current_ua=103040,power_uw=521592
2021-09-23 18:46:34,138 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:35,194 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:46:36,251 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5048,current_ua=91200,power_uw=459368
2021-09-23 18:46:37,307 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:46:38,363 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:46:39,419 # powermon,channel=Nucleo-f767zi shunt_uv=46680,bus_mv=5080,current_ua=93360,power_uw=472440
2021-09-23 18:46:40,475 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:41,531 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:46:42,586 # powermon,channel=Nucleo-f767zi shunt_uv=48520,bus_mv=5064,current_ua=97040,power_uw=491208
2021-09-23 18:46:43,643 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:44,699 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:46:45,754 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:46,811 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:46:47,867 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:46:48,923 # powermon,channel=Nucleo-f767zi shunt_uv=48800,bus_mv=5072,current_ua=97600,power_uw=497056
2021-09-23 18:46:49,979 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5032,current_ua=91280,power_uw=457912
2021-09-23 18:46:51,035 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:46:52,092 # powermon,channel=Nucleo-f767zi shunt_uv=46640,bus_mv=5064,current_ua=93280,power_uw=470952
2021-09-23 18:46:53,147 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5040,current_ua=91200,power_uw=458640
2021-09-23 18:46:54,203 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:55,259 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:56,315 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:46:57,371 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:46:58,428 # powermon,channel=Nucleo-f767zi shunt_uv=50440,bus_mv=5072,current_ua=100880,power_uw=512272
2021-09-23 18:46:59,483 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:47:00,539 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:01,596 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:02,651 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:03,707 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:04,763 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:05,819 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:06,875 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:47:07,932 # powermon,channel=Nucleo-f767zi shunt_uv=51920,bus_mv=5072,current_ua=103840,power_uw=527488
2021-09-23 18:47:08,987 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:10,043 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:11,100 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:47:12,156 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:13,211 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:14,267 # powermon,channel=Nucleo-f767zi shunt_uv=46160,bus_mv=5080,current_ua=92320,power_uw=467360
2021-09-23 18:47:15,323 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:16,379 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:17,436 # powermon,channel=Nucleo-f767zi shunt_uv=49280,bus_mv=5064,current_ua=98560,power_uw=501336
2021-09-23 18:47:18,491 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:19,547 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:47:20,603 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5040,current_ua=91200,power_uw=458640
2021-09-23 18:47:21,659 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:22,716 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:23,772 # powermon,channel=Nucleo-f767zi shunt_uv=48760,bus_mv=5072,current_ua=97520,power_uw=497056
2021-09-23 18:47:24,827 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:25,883 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:26,939 # powermon,channel=Nucleo-f767zi shunt_uv=46640,bus_mv=5064,current_ua=93280,power_uw=470952
2021-09-23 18:47:27,995 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:29,051 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:47:30,107 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:31,163 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:32,219 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:33,276 # powermon,channel=Nucleo-f767zi shunt_uv=51360,bus_mv=5072,current_ua=102720,power_uw=522416
2021-09-23 18:47:34,332 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:35,388 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:36,443 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:37,499 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:38,556 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:39,612 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:40,668 # powermon,channel=Nucleo-f767zi shunt_uv=45560,bus_mv=5072,current_ua=91120,power_uw=461552
2021-09-23 18:47:41,724 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:42,779 # powermon,channel=Nucleo-f767zi shunt_uv=50960,bus_mv=5072,current_ua=101920,power_uw=517344
2021-09-23 18:47:43,835 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:44,892 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:45,947 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:47,003 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:48,060 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:49,116 # powermon,channel=Nucleo-f767zi shunt_uv=46120,bus_mv=5080,current_ua=92240,power_uw=467360
2021-09-23 18:47:50,172 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:47:51,228 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:52,284 # powermon,channel=Nucleo-f767zi shunt_uv=49800,bus_mv=5064,current_ua=99600,power_uw=506400
2021-09-23 18:47:53,339 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:47:54,395 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:47:55,452 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:47:56,508 # powermon,channel=Nucleo-f767zi shunt_uv=46640,bus_mv=5072,current_ua=93280,power_uw=471696
2021-09-23 18:47:57,564 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:47:58,620 # powermon,channel=Nucleo-f767zi shunt_uv=47800,bus_mv=5072,current_ua=95600,power_uw=486912
2021-09-23 18:47:59,676 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:00,732 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:01,788 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5064,current_ua=91200,power_uw=460824
2021-09-23 18:48:02,844 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:03,901 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:04,956 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:06,012 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:07,068 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:08,125 # powermon,channel=Nucleo-f767zi shunt_uv=50920,bus_mv=5072,current_ua=101840,power_uw=517344
2021-09-23 18:48:09,181 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:10,236 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:11,292 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:12,349 # powermon,channel=Nucleo-f767zi shunt_uv=51400,bus_mv=5080,current_ua=102800,power_uw=523240
2021-09-23 18:48:13,406 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:14,461 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:15,517 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:16,573 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:17,629 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:18,685 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:48:19,741 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:20,797 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:48:21,853 # powermon,channel=Nucleo-f767zi shunt_uv=49800,bus_mv=5064,current_ua=99600,power_uw=506400
2021-09-23 18:48:22,909 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:48:23,965 # powermon,channel=Nucleo-f767zi shunt_uv=46120,bus_mv=5072,current_ua=92240,power_uw=466624
2021-09-23 18:48:25,021 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:26,077 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:48:27,133 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:48:28,189 # powermon,channel=Nucleo-f767zi shunt_uv=47760,bus_mv=5080,current_ua=95520,power_uw=487680
2021-09-23 18:48:29,245 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:48:30,301 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:31,357 # powermon,channel=Nucleo-f767zi shunt_uv=47200,bus_mv=5064,current_ua=94400,power_uw=476016
2021-09-23 18:48:32,413 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:33,469 # powermon,channel=Nucleo-f767zi shunt_uv=48280,bus_mv=5072,current_ua=96560,power_uw=491984
2021-09-23 18:48:34,524 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:48:35,581 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:36,637 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:48:37,693 # powermon,channel=Nucleo-f767zi shunt_uv=50480,bus_mv=5072,current_ua=100960,power_uw=512272
2021-09-23 18:48:38,749 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:39,805 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:40,861 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:41,917 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:42,973 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:44,029 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:48:45,086 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:46,141 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:47,197 # powermon,channel=Nucleo-f767zi shunt_uv=51440,bus_mv=5072,current_ua=102880,power_uw=522416
2021-09-23 18:48:48,253 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:48:49,309 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:50,365 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:51,421 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:48:52,477 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:48:53,533 # powermon,channel=Nucleo-f767zi shunt_uv=46160,bus_mv=5080,current_ua=92320,power_uw=467360
2021-09-23 18:48:54,589 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:48:55,646 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:48:56,701 # powermon,channel=Nucleo-f767zi shunt_uv=50120,bus_mv=5064,current_ua=100240,power_uw=506400
2021-09-23 18:48:57,757 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:58,813 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:48:59,869 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:49:00,925 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:49:01,983 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:49:03,037 # powermon,channel=Nucleo-f767zi shunt_uv=48240,bus_mv=5072,current_ua=96480,power_uw=486912
2021-09-23 18:49:04,093 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:49:05,149 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:49:06,205 # powermon,channel=Nucleo-f767zi shunt_uv=47200,bus_mv=5064,current_ua=94400,power_uw=476016
2021-09-23 18:49:07,261 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5080,current_ua=91200,power_uw=462280
2021-09-23 18:49:08,318 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:49:09,374 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:49:10,429 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5072,current_ua=91280,power_uw=461552
2021-09-23 18:49:11,486 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:49:12,541 # powermon,channel=Nucleo-f767zi shunt_uv=50880,bus_mv=5072,current_ua=101760,power_uw=517344
2021-09-23 18:49:13,597 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:49:14,653 # powermon,channel=Nucleo-f767zi shunt_uv=45640,bus_mv=5080,current_ua=91280,power_uw=462280
2021-09-23 18:49:15,709 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:49:16,765 # powermon,channel=Nucleo-f767zi shunt_uv=45600,bus_mv=5072,current_ua=91200,power_uw=461552
2021-09-23 18:49:17,822 # powermon,channel=Nucleo-f767zi shunt_uv=45560,bus_mv=5080,current_ua=91120,power_uw=462280
2021-09-23 18:49:18,001 # Exiting Pyterm
```

</p>

</details> 

```
cat regulator_off.txt | tr ',' ' ' | awk '{print $9}' | sed 's/current_ua=//g' | awk '{x+=$0}END{print x/NR}'
92084.8
```

### Result
The difference between the average current consumptions of (A) and (B) is  `92250.5 uA - 92084.8 uA = 165.7 uA`. To conclude from the results, the influence of the regulator when the CPU is not in standby is not significant.

Reply:
I will also remeasure with a multimeter as soon I have one at hand.

Reply:
So if I am not mistaken, to measure the current consumption on the `nucleo` boards,
it is as simple as removing a jumper and connect a multimeter to the `IDD` pins. For the `nucleo-f767zi` it is Jumper 5. (And do `ETH->MACPMTCSR |= 1` to power down Ethernet. )
[datasheet section 6.7](https://www.st.com/resource/en/data_brief/nucleo-f767zi.pdf)
When the regulator is switched on explicitly I get at most `34.30 mA` in normal operation mode
And when the regulator is not explicitly enabled I get at most `34.19 mA` in normal operation mode.

So, if at all, the difference is about `110 µA`.
How do we conclude? Keep the regulator enabled by default all the time?
If you think it is worth to disable it, then I must get a coin cell and remove a solder bridge to know what is going on, when the board is powered from `VBat`, because it could be that the regulator must be turned on all the time in that case anyways.

Reply:
So there is a difference. You already have code in place to deal with turning off the regulator and save that bit of power - so just keep it.

I just wanted to make sure we don't keep code around to chase a phantom.